Carbon.xcodeproj/project.pbxproj

```diff
@@ -655,6 +655,7 @@
 					"@executable_path/Frameworks",
 					"@loader_path/Frameworks",
 				);
+				MARKETING_VERSION = 0.3;
```

Comment:
Is this a necessary change?

Reply:
I think this is a Xcode 11 thing. I did not notice it when I committed. Should I remove it?

Reply:
It should be erased if it works properly without it, IMHO.